src/Aspire.Hosting.Elasticsearch/ElasticsearchResource.cs

```diff
@@ -26,6 +26,7 @@ public class ElasticsearchResource : ContainerResource, IResourceWithConnectionS
     /// <param name="password">A parameter that contains the Elasticsearch superuser password.</param>
     public ElasticsearchResource(string name, ParameterResource password) : base(name)
     {
+        ArgumentNullException.ThrowIfNull(name);
```

Comment:
ContainerResource has name checking. 
And since we do not use the name in the bottom context, checking here for 'null' is not needed at the moment. If in the future, in this constructor, the use of a name appears, then the need for 'ThrowIsNull' will appear